src/index.js

```diff
@@ -13,3 +13,4 @@ module.exports.isAuthenticated = isAuthenticated;
 module.exports.isAuthorized = isAuthorized;
 module.exports.Redis = require('./redis').Redis;
 module.exports.Elastic = require('./elastic').Elastic;
+module.exports.Fetch = require('node-fetch');
```

Comment:
How come that `F` is an upper case now?  `fetch()` is always lower case (it's not a constructor).  Let's use `.fetch` (lowercase).